kie-server-parent/kie-server-common/src/main/java/org/kie/server/common/KeyStoreHelperUtil.java

```diff
@@ -7,14 +7,44 @@
 import org.slf4j.LoggerFactory;
 
 public class KeyStoreHelperUtil {
+    // the private key identifier for REST controller
+    private static final String PROP_PWD__REST_ALIAS = "kie.keystore.key.rest.alias";
+    // the private key identifier for REST controller
+    private static final String PROP_PWD__REST_PWD = "kie.keystore.key.rest.pwd";
+
+    // the private key identifier for controller
+     private static final String PROP_PWD_CTRL_ALIAS = "kie.keystore.key.ctrl.alias";
+    // the private key identifier for controller
+    private static final String PROP_PWD_CTRL_PWD = "kie.keystore.key.ctrl.pwd";
+
     private static final Logger logger = LoggerFactory.getLogger(KeyStoreHelperUtil.class);
 
-    public static String loadPassword(KieServerConfig config) {
+    public static String loadRestControllerPassword() {
```

Comment:
this is not really controller password but kie server 

in general kie server can call controller and here is the password for controller required - meaning one that is configured in controller's security domain). Then controller can also call kie servers and here the password for kie server is required - meaning one that is configured in kie server's security domain.

So I'd rename the method to loadServerPassword and loadControllerPassword